Screenbox/Pages/PlayerPage.xaml

```diff
@@ -745,41 +781,47 @@
 
             <VisualStateGroup x:Name="LayoutGroup" CurrentStateChanged="OnLayoutVisualStateChanged">
                 <VisualState x:Name="Normal" />
+
                 <VisualState x:Name="Fullscreen">
                     <VisualState.Setters>
                         <Setter Target="HeaderBackground.Height" Value="NaN" />
+                        <Setter Target="TitleBarElement.(Grid.Column)" Value="2" />
+                        <Setter Target="TitleBarElement.(Grid.ColumnSpan)" Value="4" />
                         <Setter Target="BackButton.Visibility" Value="Collapsed" />
                         <Setter Target="PlayQueueButton.Margin" Value="20,20,6,20" />
-                        <Setter Target="LivelyOptionsButton.Margin" Value="20,-16,20,0" />
+                        <Setter Target="LivelyOptionsButton.Margin" Value="6,20,6,20" />
                         <Setter Target="PlayerControls.Padding" Value="20" />
-                        <Setter Target="VideoView.ButtonMargin" Value="0,70,0,104" />
+                        <Setter Target="VideoView.ButtonMargin" Value="0,72,0,104" />
                     </VisualState.Setters>
                     <Storyboard>
                         <RepositionThemeAnimation FromVerticalOffset="-120" TargetName="AlbumArtPanel" />
                         <RepositionThemeAnimation FromVerticalOffset="-120" TargetName="PlayerControls" />
                     </Storyboard>
                 </VisualState>
+
                 <VisualState x:Name="CompactOverlay">
                     <VisualState.Setters>
                         <Setter Target="HeaderBackground.Height" Value="NaN" />
-                        <Setter Target="TitleBarElement.Height" Value="32" />
-                        <Setter Target="TitleBarElement.Margin" Value="0" />
+                        <Setter Target="TitleBarElement.(Grid.Column)" Value="3" />
+                        <Setter Target="TitleBarElement.(Grid.ColumnSpan)" Value="3" />
```

Comment:
Same grid change here as well. I don't think they really do anything.

Reply:
And for the Miniplayer visual state:

|Padding|Back Button|Queue Button|Title Text|Lively Button|Padding|
|-|-|-|-|-|-|
|Reserved|Collapsed|Collapsed|Visible|Collapsed|Reserved|